tests/test_var.py

```diff
@@ -30,13 +30,13 @@
 
 
 @pytest.mark.parametrize("theta, phi", list(zip(THETA, PHI)))
-@pytest.mark.parametrize("shots", [8192])
+@pytest.mark.parametrize("shots", [None])
```

Comment:
Do we need to parametrize over `None` (which is the default)?

Reply:
No, but in this PR rather changed the analytic argument rather than adjust how the testing is done